diff --git a/package/musl/Config.in b/package/musl/Config.in
index c1a10bdd58..fe9a5bcf8b 100644
--- a/package/musl/Config.in
+++ b/package/musl/Config.in
@@ -14,6 +14,7 @@ config BR2_PACKAGE_MUSL_ARCH_SUPPORTS
 	default y if BR2_powerpc64
 	default y if BR2_powerpc64le
 	default y if BR2_RISCV_64
+	default y if BR2_s390x
 	default y if BR2_sh
 	default y if BR2_x86_64
 	depends on !BR2_powerpc_SPE # not supported, build breaks
