--- buildroot.orig/toolchain/toolchain-buildroot/Config.in
+++ buildroot/toolchain/toolchain-buildroot/Config.in
@@ -85,7 +85,7 @@
 	depends on BR2_aarch64	   || BR2_arm   || BR2_armeb   || BR2_i386 || \
 		   BR2_microblaze  || BR2_mips  || BR2_mipsel  || BR2_mips64 || \
 		   BR2_mips64el    || BR2_or1k  || BR2_powerpc || BR2_powerpc64 || \
-		   BR2_powerpc64le || BR2_RISCV_64 || BR2_sh   || BR2_x86_64
+		   BR2_powerpc64le || BR2_RISCV_64 || BR2_sh   || BR2_x86_64 || BR2_s390x
 	depends on !BR2_powerpc_SPE # not supported, build breaks
 	depends on !(BR2_powerpc64 || BR2_powerpc64le) || BR2_POWERPC_CPU_HAS_ALTIVEC
 	# sh2 nommu is supported by musl, but we don't have support
