--- buildroot.orig/package/iproute2/Config.in
+++ buildroot/package/iproute2/Config.in
@@ -2,6 +2,7 @@
 	bool "iproute2"
 	depends on BR2_USE_MMU # fork()
 	depends on BR2_TOOLCHAIN_HEADERS_AT_LEAST_3_4 # __kernel_{u,}long_t
+	select BR2_PACKAGE_LINUX_HEADERS
 	help
 	  Kernel routing and traffic control utilities. Provides things
 	  like ip and tc.
--- buildroot.orig/package/iproute2/iproute2.mk
+++ buildroot/package/iproute2/iproute2.mk
@@ -7,7 +7,7 @@
 IPROUTE2_VERSION = 5.7.0
 IPROUTE2_SOURCE = iproute2-$(IPROUTE2_VERSION).tar.xz
 IPROUTE2_SITE = $(BR2_KERNEL_MIRROR)/linux/utils/net/iproute2
-IPROUTE2_DEPENDENCIES = host-bison host-flex host-pkgconf \
+IPROUTE2_DEPENDENCIES = host-bison host-flex host-pkgconf linux-headers \
 	$(if $(BR2_PACKAGE_LIBMNL),libmnl)
 IPROUTE2_LICENSE = GPL-2.0+
 IPROUTE2_LICENSE_FILES = COPYING
