diff --git a/package/nfs-utils/nfs-utils.mk b/package/nfs-utils/nfs-utils.mk
index ed205a26b1..73561dff3a 100644
--- a/package/nfs-utils/nfs-utils.mk
+++ b/package/nfs-utils/nfs-utils.mk
@@ -14,6 +14,9 @@ NFS_UTILS_CPE_ID_VENDOR = linux-nfs
 NFS_UTILS_AUTORECONF = YES
 
 NFS_UTILS_CONF_ENV = knfsd_cv_bsd_signals=no
+# getrpcby{number,name} are only provided if 'GQ' is defined
+NFS_UTILS_CONF_ENV += CFLAGS="$(TARGET_CFLAGS) -DGQ"
+HOST_NFS_UTILS_CONF_ENV += CFLAGS="$(TARGET_CFLAGS) -DGQ"
 
 NFS_UTILS_CONF_OPTS = \
 	--enable-tirpc \
