--- buildroot.orig/package/nfs-utils/nfs-utils.mk
+++ buildroot/package/nfs-utils/nfs-utils.mk
@@ -14,6 +14,9 @@
 NFS_UTILS_AUTORECONF = YES
 
 NFS_UTILS_CONF_ENV = knfsd_cv_bsd_signals=no
+# getrpcby{number,name} are only provided if 'GQ' is defined
+NFS_UTILS_CONF_ENV += CFLAGS="$(TARGET_CFLAGS) -DGQ"
+HOST_NFS_UTILS_CONF_ENV += CFLAGS="$(TARGET_CFLAGS) -DGQ"
 
 NFS_UTILS_CONF_OPTS = \
 	--disable-nfsv4 \
@@ -35,7 +38,7 @@
 	--without-tcp-wrappers \
 	--with-statedir=/run/nfs \
 	--disable-caps \
-	--disable-tirpc \
+	--enable-tirpc \
 	--without-systemd \
 	--with-rpcgen=internal
 HOST_NFS_UTILS_DEPENDENCIES = host-pkgconf host-libtirpc
