--- buildroot.orig/package/libtirpc/libtirpc.mk
+++ buildroot/package/libtirpc/libtirpc.mk
@@ -11,20 +11,19 @@
 LIBTIRPC_LICENSE_FILES = COPYING
 LIBTIRPC_CPE_ID_VENDOR = libtirpc_project
 
-LIBTIRPC_DEPENDENCIES = host-nfs-utils
 LIBTIRPC_INSTALL_STAGING = YES
-LIBTIRPC_AUTORECONF = YES
+LIBTIRPC_INSTALL_TARGET = NO
+LIBTIRCP_AUTORECONF = YES
+ 
+LIBTIRPC_DEPENDENCIES = host-libtirpc host-pkgconf
+HOST_LIBTIRPC_DEPENDENCIES = host-pkgconf libcap
 
 # getrpcby{number,name} are only provided if 'GQ' is defined
 LIBTIRPC_CONF_ENV = CFLAGS="$(TARGET_CFLAGS) -DGQ"
-
+HOST_LIBTIRPC_CONF_ENV = CFLAGS="$(TARGET_CFLAGS) -I$(wildcard $(CONFIG_DIR)/output/per-package/libcap/host/*/sysroot/usr/include) -DGQ"
+ 
 LIBTIRPC_CONF_OPTS = --disable-gssapi
-
-define HOST_LIBTIRPC_INSTALL_CMDS
-	$(INSTALL) -D -m 0644 $(@D)/tirpc/rpc/types.h $(HOST_DIR)/include/rpc/types.h
-	$(INSTALL) -D -m 0644 $(@D)/tirpc/netconfig.h $(HOST_DIR)/include/netconfig.h
-endef
-
+HOST_LIBTIRPC_CONF_OPTS = --disable-gssapi
+ 
 $(eval $(autotools-package))
-# We are only copying headers; no need for the autotools infrastructure
-$(eval $(host-generic-package))
+$(eval $(host-autotools-package))
