diff --git a/package/libtirpc/libtirpc.mk b/package/libtirpc/libtirpc.mk
index 179adc97d0..4cf5c92ede 100644
--- a/package/libtirpc/libtirpc.mk
+++ b/package/libtirpc/libtirpc.mk
@@ -12,9 +12,13 @@ LIBTIRPC_LICENSE_FILES = COPYING
 LIBTIRPC_CPE_ID_VENDOR = libtirpc_project
 
 LIBTIRPC_INSTALL_STAGING = YES
+LIBTIRPC_INSTALL_TARGET = NO
+
+LIBTIRPC_DEPENDENCIES = host-libtirpc host-pkgconf
+HOST_LIBTIRPC_DEPENDENCIES = host-pkgconf libcap
 
 # getrpcby{number,name} are only provided if 'GQ' is defined
-LIBTIRPC_CONF_ENV = CFLAGS="$(TARGET_CFLAGS) -DGQ"
+LIBTIRPC_CONF_ENV = CFLAGS="$(TARGET_CFLAGS) -I$(wildcard $(CONFIG_DIR)/output/per-package/libcap/host/*/sysroot/usr/include) -DGQ"
 
 ifeq ($(BR2_PACKAGE_LIBTIRPC_GSS),y)
 LIBTIRPC_CONF_ENV += KRB5_CONFIG=$(STAGING_DIR)/usr/bin/krb5-config
